setup.sh

```diff
@@ -383,6 +381,15 @@ function append_bashcompletion()
   safe_append "source ${libdir}/${BASH_AUTOCOMPLETE}.sh" "${HOME}/${shellrc}"
 }
 
+function append_zshcompletion()
+{
+  local shellrc="$1"
```

Comment:
Can we just hardcode `.zshrc` here and avoid this extra parameter?

Reply:
> @davidbtadokoro is it possible to expand the _kw to be used with bash autocomplete?

Do you mean adapt the ZSH completion for Bash? If I understood it right, maybe there is a way to do this and I am going to look it up.

However, before this PR, the ZSH was adapted from a Bash native completion implementation and some incompatibilities arose from this non-native approach. Did not studied the Bash completion system, but the ZSH one is really complex - the official documentation even states that "_that there are probably lots of bugs around_" [(Section 10)](https://zsh.sourceforge.io/Guide/zshguide06.html) - so this adaptation may have some caveats.

Reply:
> Can we just hardcode `.zshrc` here and avoid this extra parameter?

Yes you are right! Maybe we can also hardcode `.bashrc` in the `append_bashcompletion` function, as fwiu the `shellrc` parameter existed for the function handle both Bash and ZSH (now each has its own function). What do you think?

Reply:
> > @davidbtadokoro is it possible to expand the _kw to be used with bash autocomplete?
> 
> Do you mean adapt the ZSH completion for Bash? If I understood it right, maybe there is a way to do this and I am going to look it up.
> 
> However, before this PR, the ZSH was adapted from a Bash native completion implementation and some incompatibilities arose from this non-native approach. Did not study the Bash completion system, but the ZSH one is really complex - the official documentation even states that "_there are probably lots of bugs around_" [(Section 10)](https://zsh.sourceforge.io/Guide/zshguide06.html) - so this adaptation may have some caveats.

yeah... ignore my previous comment, just focus on zsh.